pkgs/by-name/u-/u-config-unwrapped/package.nix

```diff
@@ -0,0 +1,121 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  noLibc ? stdenv.cc.isGNU && stdenv.hostPlatform.isx86 && stdenv.hostPlatform.isLinux,
+  pkgsBuildBuild,
+  installShellFiles,
+  nix-update-script,
+}:
+
+let
+  inherit (stdenv) hostPlatform;
+  inherit (lib) optionals;
+
+  platform =
+    if noLibc then
+      (
+        with hostPlatform;
+        if isLinux && isx86_32 then
+          "linux_i686"
+        else if isLinux && isx86_64 then
+          "linux_amd64"
+        else
+          throw "unsupported system ${system} for libc‐free version"
+      )
+    else
+      "generic";
+
+  cflags =
+    [
+      "-DPKG_CONFIG_PREFIX=\"\""
+      "-DPKG_CONFIG_LIBDIR=\"\""
+      "-DPKG_CONFIG_SYSTEM_INCLUDE_PATH=\"\""
+      "-DPKG_CONFIG_SYSTEM_LIBRARY_PATH=\"\""
+    ]
+    ++ optionals noLibc (
+      [
+        "-fno-builtin"
+        "-fno-asynchronous-unwind-tables"
+        "-static"
+        "-nostdlib"
+        "-Wl,--gc-sections"
+      ]
+      ++ optionals (stdenv.cc.isGNU) [
+        # avoid generating calls to memset() and memcpy()
+        "-fno-tree-loop-distribute-patterns"
+      ]
+    );
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "u-config";
+  version = "0.33.3";
+
+  outputs = [
+    "out"
+    "man"
+  ];
+
+  src = fetchurl {
```

Comment:
Can you add a comment about `fetchurl` usage over `fetchFromGithub` as part of the bootstrapping? I think it might help in future reviews

Reply:
Will do.